validation/stratified_couette_flow/stratified_couette_flow.jl

```diff
@@ -16,6 +16,18 @@ using Oceananigans.TurbulenceClosures:
                 𝒥ᴿᴺ_forcing_function, 
                 𝒥ᴺᴺ_forcing_function
 
+
+Nxy=64
+Nz=32
+arch=CPU()
+h=1
+U_wall=1
+Re=4250
+Pr=0.7
+Ri = 0.5
+Ni=10
+end_time=1000
```

Comment:
What is going on here? Isn't there already a script for invoking the utilities in this script?